metrics/nop_reporter.go

```diff
@@ -86,47 +86,45 @@ type nopCachedCount struct {
 func (nopCachedCount) ReportCount(value int64) {
 }
 
-// NopCachedGauge is an implementation of tally.CachedGauge
+// NopCachedGauge is an implementation of tally.CachedGauge and same as other Nop types don't do anything and
+// should be used for testing purposes only.
 var NopCachedGauge tally.CachedGauge = nopCachedGauge{}
 
-type nopCachedGauge struct {
-}
+type nopCachedGauge struct{}
 
-func (nopCachedGauge) ReportGauge(value float64) {
-}
+func (nopCachedGauge) ReportGauge(value float64) {}
 
 // NopCachedTimer is an implementation of tally.CachedTimer
 var NopCachedTimer tally.CachedTimer = nopCachedTimer{}
 
-type nopCachedTimer struct {
-}
+type nopCachedTimer struct{}
 
-func (nopCachedTimer) ReportTimer(interval time.Duration) {
-}
+func (nopCachedTimer) ReportTimer(interval time.Duration) {}
 
-// NopCachedHistogram is an implementation of tally.CachedHistogram
+// NopCachedHistogram is an implementation of tally.CachedHistogram and same as other Nop types don't do anything and
+// should be used for testing purposes only.
 var NopCachedHistogram tally.CachedHistogram = nopCachedHistogram{}
 
-type nopCachedHistogram struct {
-}
+type nopCachedHistogram struct{}
 
 func (nopCachedHistogram) ValueBucket(
-	bucketLowerBound, bucketUpperBound float64,
+	bucketLowerBound float64,
```

Comment:
I know, personally, it is easier to read. Really wish go fmt will help with these things :)